.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 dist
 *.zip
 coverage
+calculator.js
```

Comment:
Might want to remove this, probably just for development

Reply:
oh whoops